pkg/core/blockchain.go

```diff
@@ -468,6 +468,8 @@ func (bc *Blockchain) init() error {
 	}
 	bc.blockHeight = bHeight
 	bc.persistedHeight = bHeight
+
+	bc.log.Info("initializing caches", zap.Uint32("blockHeight", bHeight))
```

Comment:
Not sure it's useful. Debug-level, maybe.

Reply:
Fixed. Although debug logs are always off in our production environment, so it something happens there, then the node height will still be a secrete.

Reply:
If something like #3181 happens on node start it's easy to try debug, not a lot of other messages you'll see before node dies.